### PR TITLE
Feature/REP-26856: Update scala binary version to 2.12

### PR DESCRIPTION
**Pull request by harmeetsingh0013**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -126,9 +126,9 @@
         </dependency>
         
         <dependency>
-            <groupId>com.decodified</groupId>
-            <artifactId>scala-ssh_2.11</artifactId>
-            <version>0.7.0</version>
+            <groupId>com.github.seratch.com.veact</groupId>
+            <artifactId>scala-ssh_${scalaArtifactIDVersion}</artifactId>
+            <version>0.8.0-1</version>
```
Comment (yonyyatsun) [inline pom.xml]: is this works fine with scala 2.11?

Comment (harmeetsingh0013) [reply pom.xml]: @yonyyatsun that, we need to verify. But if you look at the actual source of AWScala https://github.com/seratch/AWScala/blob/master/build.sbt, they are using the same dependency and they provide support for Scala 2.10 to 2.12.

Comment (harmeetsingh0013) [reply pom.xml]: This dependency is maintained by the same person who is maintaining AWScala. https://github.com/seratch/scala-ssh

Comment (yonyyatsun) [reply pom.xml]: @harmeetsingh0013 Ok. Just test Sanity check with 2.11 (for example Report Service)

Comment (harmeetsingh0013) [reply pom.xml]: @yonyyatsun Report Service means Dispatch Cluster?

Comment (yonyyatsun) [reply pom.xml]: @harmeetsingh0013 Yes

Comment (harmeetsingh0013) [reply pom.xml]: @yonyyatsun the problem is, on our machines ReportService is not working. Maybe you guys can help us for validation.

Comment (harmeetsingh0013) [reply pom.xml]: Hey, @yonyyatsun I executed Dispatch Cluster on port 8081, which is running fine and hit  http://localhost:8081/SanityHealthCheck service for Sanity Health Check, which returns 200 Ok. Please see attached screenshot. If something pending or need to validate some other sanity health check, please let us know.

image

**Pull request merged**